src/config.rs

```diff
@@ -397,7 +397,7 @@ cf_config!(DefaultCfConfig);
 impl Default for DefaultCfConfig {
     fn default() -> DefaultCfConfig {
         DefaultCfConfig {
-            block_size: ReadableSize::kb(64),
+            block_size: ReadableSize::kb(4),
```

Comment:
Do we need to make write cf also use 4KB block size?

Reply:
I think so, despite our previous benchmark didn't test it.